docs/setup_after_installation.md

```diff
@@ -4,53 +4,47 @@ After installing Kapowarr, you should have access to the web-ui. Kapowarr needs
 
 ## Downloading
 
-One of Kapowarr's biggest features is being able to download comics. The Settings -> Download section has all settings regarding the downloading.
+Kapowarr's biggest feature is being able to download comics. The [Settings -> Download](../settings/#download) section has all settings regarding the downloading.
 
 ### Download folder
 
-The download folder (Settings -> Download -> Download Location -> Direct Download Temporary Folder) is where all downloads are downloaded to, before they get moved to their final destination. If you run Kapowarr using Docker, leave this setting to it's default value of `/app/temp_downloads` and instead change the value of `/path/to/download_folder` in the Docker command ([reference](../installation/#docker)). If you have a manual install, you can change this value to whatever you want. It is allowed to be outside your root folders.
+The download folder is where all downloads are downloaded to, before they get moved to their final destination.  
+In most cases, the default of `/app/temp_downloads` works best. If you wish to change this, refer to [Direct Download Temporary Folder](../settings/#direct-download-temporary-folder).  
 
 ### Service preference
 
-Kapowarr has the ability to download directly from servers, but also to download from services like MediaFire and Mega. Websites like getcomics.org offer the same download via multiple services (multiple download links to download the same file, via different services). This settings determines what preference you have for each service. If multiple services are offered for the same download, Kapowarr will use this preference list to determine what service to pick (if the link of the top service doesn't work, Kapowarr falls back to the other options, in order). If you have an account for one of these services (see [Credentials](#credentials) setting), you might want to put that one at the top, to make Kapowarr take advantage of the extra features that the account offers (extra bandwidth, higher rate limit, etc.).
+If you have an account with one of the supported download services (MediaFire and Mega), set that service as the priority and add a credential for it.  
+The other services will then be used as a fallback option for if a link fails.
+
+For a full explanation, see [Service Preference](../settings/#service-preference).  
 
 ### Credentials
 
-Kapowarr has the ability to download from services like MediaFire and Mega. These services apply limits to how much you can download per day, or a download speed limit. An (paid) account for one of these services often offers higher limits. Kapowarr can take advantage of these extra features that these accounts offer. Under the credentials section, you can add credentials of accounts, which Kapowarr will use when downloading, taking advantage of the extra features.
+This only applies if you have an account with MediaFire or Mega.  
```

Comment:
Credentials for MediaFire are not (yet) supported. You can only add Mega credentials currently.

Reply:
With a service provider, you have an account with them. An account with a store, for example.

Reply:
Okay thanks. Either way, "MediaFire or Mega" should be "Mega". And that's the last thing I could find. So unless you're planning on making anymore changes, I'll merge after this conversation is resolved.

Reply:
Changed to Mega, ready to merge.